lib/process-services-cloud/src/lib/process/start-process/components/start-process-cloud.component.ts

```diff
@@ -254,13 +280,7 @@ export class StartProcessCloudComponent implements OnChanges, OnInit, OnDestroy
 
         this.startProcessCloudService
             .getProcessDefinitions(this.appName)
-            .pipe(
-                tap(() => {
-                    this.processDefinitionLoaded = true;
-                    this.loading$.next(false);
-                }),
-                takeUntil(this.onDestroy$)
-            )
+            .pipe(takeUntil(this.onDestroy$))
```

Comment:
I was wondering (when working on my PR), whether we can add take(1) / takeFirst() in this case

Reply:
I think it should work, in fact for one-time API calls it's event better than takeUntil

Reply:
but I wondering what will happen when we will replace takeUntil(this.onDestroy$) with only take(1) at the time we will lose component from the view and don't receive response yet, guessing that this subscribing will be completed only when we receive something not when component is out of the view, so probably even we will use take(1) we should still keep takeUntil(this.onDestroy$).
(gpt)
![image](https://github.com/user-attachments/assets/5846574a-a1c9-4c81-9654-20553ca9e856)

Reply:
This is actually interesting topic that you brought up. Usually I was using first() with httpClient calls (and even that is redundant as http calls usually produce one result and complete), however there are cases that it would be better to cancel the subscription with onDestroy$.
I came across this discussion on stackoverflow, I encourage you to take a look to see various edge cases people describe when unsubscribing is a better option: https://stackoverflow.com/questions/35042929/is-it-necessary-to-unsubscribe-from-observables-created-by-http-methods-to-avoid

Reply:
Thank you for reminder and ref, I just forgot that some of the Angular services can automatically unsubscribe after one value being received. Like a HttpClient does, but as it was touched in discussion, we still should relay at least on takeUntil(onDestroy$), resolved